From e79835037312f8d58c8c555e1ebf5746e09d10cd Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 7 Aug 2020 14:31:16 -0700
Subject: [PATCH] powerpc: Do not ask compiler for finding arch

This does not work well in cross compiling environments like OE
and moreover it uses its own -mcpu/-march options via cflags

Upstream-Status: Inappropriate [ OE-Specific]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 sysdeps/powerpc/preconfigure    | 5 +----
 sysdeps/powerpc/preconfigure.ac | 5 +----
 2 files changed, 2 insertions(+), 8 deletions(-)

diff --git a/sysdeps/powerpc/preconfigure b/sysdeps/powerpc/preconfigure
index 9e5a07ab6d..a308bdd863 100644
--- a/sysdeps/powerpc/preconfigure
+++ b/sysdeps/powerpc/preconfigure
@@ -29,10 +29,7 @@ esac
 # directive which shows up, and try using it.
 case "${machine}:${submachine}" in
 *powerpc*:)
-  archcpu=`echo "int foo () { return 0; }" \
-	   | $CC $CFLAGS $CPPFLAGS -S -frecord-gcc-switches -xc -o - - \
-	   | grep -E "mcpu=|.machine" -m 1 \
-	   | sed -e "s/.*machine //" -e "s/.*mcpu=\(.*\)\"/\1/"`
+  archcpu=''
   # Note if you add patterns here you must ensure that an appropriate
   # directory exists in sysdeps/powerpc.  Likewise, if we find a
   # cpu, don't let the generic configure append extra compiler options.
diff --git a/sysdeps/powerpc/preconfigure.ac b/sysdeps/powerpc/preconfigure.ac
index 14b6dafd4a..1acc6a457c 100644
--- a/sysdeps/powerpc/preconfigure.ac
+++ b/sysdeps/powerpc/preconfigure.ac
@@ -29,10 +29,7 @@ esac
 # directive which shows up, and try using it.
 case "${machine}:${submachine}" in
 *powerpc*:)
-  archcpu=`echo "int foo () { return 0; }" \
-	   | $CC $CFLAGS $CPPFLAGS -S -frecord-gcc-switches -xc -o - - \
-	   | grep -E "mcpu=|[.]machine" -m 1 \
-	   | sed -e "s/.*machine //" -e "s/.*mcpu=\(.*\)\"/\1/"`
+  archcpu=''
   # Note if you add patterns here you must ensure that an appropriate
   # directory exists in sysdeps/powerpc.  Likewise, if we find a
   # cpu, don't let the generic configure append extra compiler options.
